TECHNICAL_DETAILS.md

```diff
@@ -0,0 +1,13 @@
+## What are the upcoming changes?
+In Q2 2024, NEAR protocol will introduce the biggest upgrade since its mainnet launch in how the protocol operates, Stateless validation. The upgrade includes the following features:
+* Stateless validation: Validators do not have to track all shards anymore and can retrieve everything they need to validate state change (state witness) over the network. With stateless validation, NEAR can finally achieve true form of sharding and hardware requirements for a validator node can be dramatically reduced. State validation does not require local tracking of any shards as nodes can retrieve everything they need to validate state change over the network. As a result, hardware requirements for a validator node can be dramatically reduced.
+* In memory trie: Validator nodes can load the entire state into RAM for maximized performance. With minimized storage access, transaction throughput will be greatly improved.
```

Comment:
The "greatly" is still under evaluation. In-memory-trie reduces the storage access latency but then other things become bottlenecks and the total improvement isn't as great as just the `memory speed` / `disk speed`

Reply:
We emphasise our intentions here, I think such wording is fine